marconi-cardano-indexers/test/Spec/Marconi/Cardano/DbSyncComparison.hs

```diff
@@ -13,37 +16,75 @@ tests =
-    , mkBlockInfoQueryBySlotNoTest "At slot number 43372792" Mainnet Alonzo1 43372792
-    , mkBlockInfoQueryBySlotNoTest "At slot number 43372972" Mainnet Alonzo2 43372972
-    , mkBlockInfoQueryBySlotNoTest "At slot number 45748924" Mainnet Alonzo2 45748924
-    , mkBlockInfoQueryBySlotNoTest "At slot number 48124747" Mainnet Alonzo2 48124747
-    , mkBlockInfoQueryBySlotNoTest "At slot number 72316896" Mainnet Babbage1 72316896
-    , mkBlockInfoQueryBySlotNoTest "At slot number 77064584" Mainnet Babbage1 77064584
-    , mkBlockInfoQueryBySlotNoTest "At slot number 77068768" Mainnet Babbage1 77068768
-    , mkBlockInfoQueryBySlotNoTest "At slot number 84844885" Mainnet Babbage2 84844885
-    , mkBlockInfoQueryBySlotNoTest "At slot number 87198551" Mainnet Babbage2 87198551
-    , mkBlockInfoQueryBySlotNoTest "At slot number 89596758" Mainnet Babbage2 89596758
+    [ mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Byron1 0
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Byron1 6
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Byron1 206938
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Byron2 3801600
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Byron2 3901600
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Byron2 4039199
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Shelley 4492800
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Shelley 4520900
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Shelley 9244771
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Allegra 16588800
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Allegra 18698800
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Allegra 21340776
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Mary 23068800
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Mary 25168800
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Mary 27820794
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Alonzo1 39916975
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Alonzo1 43372584
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Alonzo1 43372792
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Alonzo2 43372972
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Alonzo2 45748924
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Alonzo2 48124747
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Babbage1 72316896
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Babbage1 77064584
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Babbage1 77068768
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Babbage2 84844885
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Babbage2 87198551
+    , mkBlockInfoQueryBySlotNoTest $ blockInfoConfig Mainnet Babbage2 89596758
```

Comment:
My reasoning behind not adding a `blockInfoConfig` was that I'd like the test name to be easily modifiable for each test in particular. This is very useful if I want to just run one test based on its name. Unless you know a better way to do this, I wouldn't make this modification here.

Reply:
Thanks for this. I made test name a parameter again, which I agree is a good idea.